### DISCUSSION TITLE
Query database columns

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#544] [Q&A]
When using a dataview query to create the table I have noticed the YAML information from the resulting pages don't appear as suggested columns in the drop-down.

I don't want to add a column that is already in the page, am I missing something?

**Comment by RafaelGB** [comment]
Its a known problem with a `help wanted` tag https://github.com/RafaelGB/obsidian-db-folder/issues/377

With dv query we are already filtering all the columns that are not specified by the table, so we can not obtain that info.

I think we can use the LIST mode of Dataview to obtain all the existing columns, but requires some investigation
